cmd/utils/flags.go

```diff
@@ -330,6 +330,11 @@ var (
 		Value:  512,
 		EnvVar: "KLAYTN_SNAPSHOT_CACHE_SIZE",
 	}
+	SnapshotGenAsync = cli.BoolTFlag{
+		Name:   "snapshot.gen.async",
```

Comment:
hmm.. is there any arguments starts with `snapshot.gen`?
I am not sure the parameter is good with this. what about `snapshot.async-gen`?

Reply:
Renamed `GenAsync` to `AsyncGen` as well.